core/object/message_queue.cpp

```diff
@@ -93,7 +95,7 @@ Error CallQueue::push_callablep(const Callable &p_callable, const Variant **p_ar
 
 	if ((page_bytes[pages_used - 1] + room_needed) > uint32_t(PAGE_SIZE_BYTES)) {
 		if (pages_used == max_pages) {
-			ERR_PRINT("Failed method: " + p_callable + ". Message queue out of memory. " + error_text);
+			printf((const char*)("Failed method: " + p_callable + ". Message queue out of memory. " + error_text).get_data());
```

Comment:
Also this cast isn't valid, it casts `const char32_t *` to `const char *`

Reply:
Passing dynamic value to the `printf` will cause warnings with some compilers (it's not safe since string can contain `%*`), use something like:

```c++
printf("%s\n", ("Failed method: " + p_callable + ". Message queue out of memory. " + error_text).utf8().get_data());
```

or 

```c++
printf("Failed method: %s. Message queue out of memory. %s\n", String(p_callable).utf8().get_data(), error_text.utf8().get_data());
```

Reply:
That does seem a better approach

Reply:
> Also this cast isn't valid, it casts const char32_t * to const char *, I think it should be:

It's `.utf8().get_data()` not just `.utf8()` (which returns `CharString`object).

Reply:
That's what the cast was for 🙂, it has:
```cpp
operator const char *() const { return get_data(); };
```